docs/root/configuration/overview/v2_overview.rst

```diff
@@ -590,7 +590,7 @@ Management Server has a statistics tree rooted at *control_plane.* with the foll
    :header: Name, Type, Description
    :widths: 1, 1, 2
 
-   connected_state, Gauge, A boolean (1 for connected and 0 for disconnected) that indicates the current connection state with management server
+   connected_state, BoolIndicator, Current connection state with management server
```

Comment:
looks like you might have accidentally deleted part of the sentence?

Reply:
I'm fine either way, but please be consistent in lds.rst above.

Reply:
These are different columns in a table - the commas are delimiters.

Matt, by consistent do you mean control_plane.connected_state vs just connected_state? If so, that difference was already there, so I left it as is - is it better to have the "control_plane." prefix, or no prefix?